main.rb

```diff
@@ -0,0 +1,6 @@
+class Hello
+
+  def say_hello
+    puts "hello world
```

Comment:
Looks like you are missing the closing double quote here to end the string literal ( "hello world" )